ui-tests/__test__/flows/navigate-prompts/stay-on-current-prompt.ts

```diff
@@ -0,0 +1,28 @@
+import { Page } from 'playwright/test';
+import { getSelector, waitForAnimationEnd } from '../../helpers';
+import testIds from '../../../../src/helper/test-ids';
+import { closeTab } from '../close-tab';
+import { openNewTab } from '../open-new-tab';
+
+export const stayOnCurrentPrompt = async (page: Page, skipScreenshots?: boolean): Promise<void> => {
+  await closeTab(page, false, true);
+  await openNewTab(page, false, true);
+
+  // Write prompt without sending it
+  await page.locator(`${getSelector(testIds.prompt.input)}`).fill('This is the first unsent user prompt');
+  await waitForAnimationEnd(page);
+
+  let promptInput = await page.locator(`${getSelector(testIds.prompt.input)}`);
```

Comment:
nit: define this selector for the prompt input as a `const`, instead of having duplicated code. Actually, I think you only have to define the locator once, and then this locator will keep updating. Try to see if that works.

Reply:
Good idea, turns out the wrong screenshot height was related to defining the promptInput multiple times, as addressing this comment also fixed the screenshot height problem

Reply:
Actually I found out that that the two issues are not correlated, the fact that I got shorter screenshots is due to the fact that I reran only the up/down arrow navigation e2e tests.
(https://github.com/aws/mynah-ui/pull/205#issuecomment-2547957582)